src/main/kotlin/io/github/mojira/arisa/modules/DuplicateMessageModule.kt

```diff
@@ -94,9 +94,12 @@ class DuplicateMessageModule(
             getField(this[0])
         }
 
-    private fun getPrivateMessageOrNull(issues: List<Issue>): String? {
+    private fun getPrivateMessageOrNull(child: Issue, issues: List<Issue>): String? {
         val parentSecurity = issues.getCommonFieldOrNull { it.securityLevel }
-        return privateMessage.takeIf { parentSecurity != null }
+        val reporters = issues.getCommonFieldOrNull { it.reporter }
+        return privateMessage.takeIf {
+            parentSecurity != null && (reporters == null || child.reporter != reporters)
```

Comment:
Simply `reporters == null` should be enough iirc. The field will be `null` when the reporters on those ticket are not the same.